src/incubator/WheelPicker/index.tsx

```diff
@@ -78,6 +78,10 @@ export interface WheelPickerProps {
    */
   separatorsStyle?: ViewStyle;
   testID?: string;
+  /**
+   * Change the default (white) tint color of the fade view.
+   */
+  faderTintColor?: string;
```

Comment:
Hi @Asgaroth 
I'd suggest instead to add a more general prop like `faderProps` so in case we'll want to support more customizations to the Fader it will be possibly to pass all through a single prop instead of multiple props.

Reply:
Thank you, sorry for the back and forth, I should have done this from the beginning.